tests/scripts/deploy_admission_controller.sh

```diff
@@ -1,26 +1,60 @@
 #!/usr/bin/env bash
 # deploy_admission_controller.sh
 # Sets up the environment for the admission controller webhook in the active cluster.
-set -eo pipefail
+set -eEo pipefail
+
+function cleanup() {
+  set +e
+  kubectl -n rook-ceph delete validatingwebhookconfigurations $WEBHOOK_CONFIG_NAME
+  kubectl -n rook-ceph delete certificate rook-admission-controller-cert
+  kubectl -n rook-ceph delete issuers selfsigned-issuer
+  kubectl delete -f https://github.com/jetstack/cert-manager/releases/download/$CERT_VERSION/cert-manager.yaml
+  set -e
+}
+trap cleanup SIGINT ERR
 
 # Set our known directories and parameters.
 BASE_DIR=$(cd "$(dirname "$0")"; pwd)
+CERT_VERSION="v1.2.0"
```

Comment:
This is going to easily get out of date. I'd rather just point them to install cert manager according to their latest docs so that we don't manage this version. But this works for now.